### PR TITLE
Give another solution of problem #66.

### PR DESCRIPTION
**Pull request by Fisher-Wang**
Give another solution of problem 66, and fix the old solution as #142 did.

### CODE REVIEW

**Comment by rougier** [comment]
I think your solution is better because we can also use it to extract the unique colors. Could your fully replace the current solution with yours (and maybe set the difficulty level to 2 stars)?

**Comment by Fisher-Wang** [comment]
Sure. I've replace the solution with mine and set the difficulty in all the relevant file. Thanks for your appreciation.

File: source/exercises100.ktx [hunk 1]
```diff
@@ -855,22 +855,12 @@ F = np.bincount(I,X)
...
-I = np.random.randint(0,2,(h,w,3)).astype(np.ubyte)
-# The parenthesis around 256*256 ensure the dtpye promoted to uint32 properly.Otherwise the first term will overflow.
-F = I[...,0]*(256*256) + I[...,1]*256 +I[...,2]
-n = len(np.unique(F))
-print(np.unique(I))
-
-# Another solution
 # Author: Fisher Wang
 
 w, h = 16, 16
```
Comment (rougier) [inline source/exercises100.ktx:866]: I instead of img (just to be consistent with other notations).

Comment (rougier) [inline source/exercises100.ktx:866]: w, h = 256, 256
img = np.random.randint(0, 4 (w, h, 3)).astype(np.ubyte)

This wil "minimize" the number of unique colors. If you use 256 values for a 16x16 image, chances are that all colos are unique.

**Comment by rougier** [comment]
Thanks.

**Pull request merged**